testutil/eth2/deposit_data.go

```diff
@@ -0,0 +1,79 @@
+package eth2
```

Comment:
@xenowits why is this in testutil? isn't it used in the normal DKG process?

Reply:
I asked him to put it here for now, next to `keystore`, we need to move them both and some other "eth2" related utils somewhere else but I'm not sure where. Waiting for a pattern to emerge.

@xenowits please rename package to `depositdata`

Reply:
@corverroos don't you think we should avoid the eth2 term as it has become unofficial? Perhaps instead of using eth2 we could use ECL?

Reply:
as discussed, we are sticking with eth2 for now